Libraries/LibJS/Bytecode/Instruction.h

```diff
@@ -172,6 +179,9 @@
     O(ThrowIfNotObject)                \
     O(ThrowIfNullish)                  \
     O(ThrowIfTDZ)                      \
+    O(ToBoolean)                       \
```

Comment:
Couldn't this technically be done with `!!`
(alternatively we could now peephole `!!` to this)